### PR TITLE
Build XyceWrapper for Julia v1.11

### PR DESCRIPTION
**Pull request by staticfloat**

### CODE REVIEW

File: X/XyceWrapper/build_tarballs.jl [hunk 1]
```diff
@@ -2,10 +2,10 @@
 # `julia build_tarballs.jl --help` to see a usage message.
 using BinaryBuilder, Pkg
 
-julia_versions = [v"1.6.3", v"1.7", v"1.8", v"1.9", v"1.10"]
+julia_versions = [v"1.7", v"1.8", v"1.9", v"1.10", v"1.11"]
```
Comment (fingolfin) [inline X/XyceWrapper/build_tarballs.jl:5]: This also drop 1.6. Intentionally?

Comment (giordano) [reply X/XyceWrapper/build_tarballs.jl:5]: But in principle nothing should have changed since last build, right?

Comment (staticfloat) [reply X/XyceWrapper/build_tarballs.jl:5]: As usual, I am investigator, victim and perpetrator: https://github.com/JuliaPackaging/Yggdrasil/pull/6180/files

The issue isn't Julia v1.6, it's `aarch64` linux.

Comment (staticfloat) [reply X/XyceWrapper/build_tarballs.jl:5]: It would be nice to have a way to tell BB "filter my own platforms to match this dependency"

**Comment by giordano** [comment]
And compat entries would probably be good anyway :slightly_smiling_face:

**Comment by staticfloat** [comment]
This PR has _not_ been as smooth as I thought it would be.

**Pull request merged**